### PR TITLE
Add instructions to open the .dapr folder in the user profile on Windows

### LINKED ISSUE #3939: Clarify instructions to open the .dapr folder in the user profile on Windows
**Describe the issue**
Under **Initialize Dapr in your local environment** in **Step 5: Verify components directory has been initialized**:

The instruction to open the `.dapr` folder in the user profile for Windows only works when the Command Prompt (cmd) is used and the instruction does not work with PowerShell (it points to the Documents folder). PowerShell is usually the default prompt when Windows Terminal is used so the current instruction could often lead to the incorrect folder.

**URL of the docs**
https://docs.dapr.io/getting-started/install-dapr-selfhost/

**Expected content**
Clarify the instruction for both cmd and PowerShell by adding explicit instructions for the command prompt and PowerShell:

cmd: explorer "%USERPROFILE%\.dapr\"

PowerShell: explorer  "$env:USERPROFILE\.dapr\"

### PR DESCRIPTION
**Pull request by prashantrewar**
Thank you for helping make the Dapr documentation better!

**Please follow this checklist before submitting:**
- [x] Commits are signed with Developer Certificate of Origin (DCO - [learn more](https://docs.dapr.io/contributing/contributing-overview/#developer-certificate-of-origin-signing-your-work))
- [x] [Read the contribution guide](https://docs.dapr.io/contributing/docs-contrib/contributing-docs/)
- [x] Commands include options for Linux, MacOS, and Windows within codetabs
- [ ] New file and folder names are globally unique
- [ ] Page references use shortcodes instead of markdown or URL links
- [ ] Images use HTML style and have alternative text
- [x] Places where multiple code/command options are given have codetabs

In addition, please fill out the following to help reviewers understand this pull request:

## Issue reference
Fixes #3939

### CODE REVIEW

File: daprdocs/content/en/getting-started/install-dapr-selfhost.md [hunk 1]
```diff
@@ -137,7 +137,11 @@ ls $HOME/.dapr
 {{% codetab %}}
```
Comment (msfussell) [inline daprdocs/content/en/getting-started/install-dapr-selfhost.md]:
```suggestion
Powershell
```

File: daprdocs/content/en/getting-started/install-dapr-selfhost.md [hunk 2]
```diff
@@ -137,7 +137,11 @@ ls $HOME/.dapr
...
 
 ```powershell
-explorer "%USERPROFILE%\.dapr\"
+explorer "$env:USERPROFILE.dapr"
+```
+
```
Comment (msfussell) [inline daprdocs/content/en/getting-started/install-dapr-selfhost.md:142]:
```suggestion
command line
```

File: daprdocs/content/en/getting-started/install-dapr-selfhost.md (hunk 1 above)
Comment (msfussell) [reply daprdocs/content/en/getting-started/install-dapr-selfhost.md:138]: Suggest being explicit with the commands

**Review by msfussell — COMMENTED** [review]
Suggest being explicit with the commands with some text

**Review by msfussell — COMMENTED** [review]
Suggest being explicit with the commands with text

File: daprdocs/content/en/getting-started/install-dapr-selfhost.md (hunk 2 above)
Comment (prashantrewar) [reply daprdocs/content/en/getting-started/install-dapr-selfhost.md:142]: Done!

File: daprdocs/content/en/getting-started/install-dapr-selfhost.md (hunk 1 above)
Comment (prashantrewar) [reply daprdocs/content/en/getting-started/install-dapr-selfhost.md]: Done!

File: daprdocs/content/en/getting-started/install-dapr-selfhost.md [hunk 3]
```diff
@@ -135,9 +135,13 @@ ls $HOME/.dapr
 {{% /codetab %}}
 
 {{% codetab %}}
-
+Powershell
```
Comment (hhunter-ms) [inline daprdocs/content/en/getting-started/install-dapr-selfhost.md]:
```suggestion
You can verify using either PowerShell or command line. If using PowerShell, run:
```

File: daprdocs/content/en/getting-started/install-dapr-selfhost.md [hunk 4]
```diff
@@ -135,9 +135,13 @@ ls $HOME/.dapr
...
 ```powershell
-explorer "%USERPROFILE%\.dapr\"
+explorer "$env:USERPROFILE.dapr"
+```
+command line
```
Comment (hhunter-ms) [inline daprdocs/content/en/getting-started/install-dapr-selfhost.md]:
```suggestion

If using command line, run: 
```

**Review by hhunter-ms — CHANGES_REQUESTED** [review]
just clarifying a bit

File: daprdocs/content/en/getting-started/install-dapr-selfhost.md (hunk 3 above)
Comment (prashantrewar) [reply daprdocs/content/en/getting-started/install-dapr-selfhost.md]: Done!

File: daprdocs/content/en/getting-started/install-dapr-selfhost.md (hunk 4 above)
Comment (prashantrewar) [reply daprdocs/content/en/getting-started/install-dapr-selfhost.md]: Done!

**Review by hhunter-ms — APPROVED** [review]

File: daprdocs/content/en/getting-started/install-dapr-selfhost.md [hunk 5]
```diff
@@ -135,9 +135,14 @@ ls $HOME/.dapr
 {{% /codetab %}}
 
 {{% codetab %}}
-
+You can verify using either PowerShell or command line. If using PowerShell, run:
 ```powershell
-explorer "%USERPROFILE%\.dapr\"
+explorer "$env:USERPROFILE.dapr"
```
Comment (marcduiker) [inline daprdocs/content/en/getting-started/install-dapr-selfhost.md]:
```suggestion
explorer "$env:USERPROFILE\.dapr"
```

File: daprdocs/content/en/getting-started/install-dapr-selfhost.md [hunk 6]
```diff
@@ -135,9 +135,14 @@ ls $HOME/.dapr
...
+```
+
+If using command line, run: 
+```cmd
+explorer "%USERPROFILE%.dapr"
```
Comment (marcduiker) [inline daprdocs/content/en/getting-started/install-dapr-selfhost.md]:
```suggestion
explorer "%USERPROFILE%\.dapr"
```

**Review by marcduiker — DISMISSED** [review]
Almost there! It was missing a backslash (I did not include that when I created the issue).

**Comment by prashantrewar** [comment]
> Almost there! It was missing a backslash (I did not include that when I created the issue).

I updated the changes. Please take a look and suggest changes if required.

Thank you @msfussell, @hhunter-ms, and @marcduiker for your suggestion and guidance.

**Pull request merged**